src/OrchardCore/OrchardCore.Email.Core/Services/SmtpService.cs

```diff
@@ -34,22 +43,33 @@ IStringLocalizer<SmtpService> stringLocalizer
             S = stringLocalizer;
         }
 
+        /// <summary>
+        /// Sends the specified message to an SMTP server for delivery.
+        /// </summary>
+        /// <param name="message">The message to be sent.</param>
+        /// <returns>A <see cref="SmtpResult"/> that holds information about the sent message, for instance if it has sent successfully or if it has failed.</returns>
+        /// <remarks>This method allows to send an email without setting <see cref="MailMessage.To"/> if <see cref="MailMessage.Cc"/> or <see cref="MailMessage.Bcc"/> is provided.</remarks>
```

Comment:
I didn't see a suitable place to mention this in our docs, so I just write here, so any developer can notice that the email can required on of the three headers